docs/dataset/advanced-usage/upload-embeddings.md

```diff
@@ -0,0 +1,65 @@
+---
+icon: kolena/classification-16
+---
+
+# :kolena-classification-16: Uploading Custom Embeddings
+
+This guide explains how to upload your own embeddings to Kolena using the Kolena SDK.
+Please ensure you have the SDK installed.
+[Instructions for installing the SDK are available here.](https://docs.kolena.com/installing-kolena/)
+
+## Step 1: Import the Embedding Upload Function
+
+To upload embeddings, use the `upload_dataset_embeddings` function from Kolena. You can import
+it with the following code:
+```
+from kolena._experimental.search import upload_dataset_embeddings
+```
+
+## Step 2: Prepare the Required DataFrame
+
+The DataFrame you upload should have two columns:
+
+- Unique Identifier Column: This is typically the locator field, which serves as a unique identifier for each entry.
+- Embedding Column: Each embedding must have the same size across all rows.
+
+### Example code
+
+Here’s an example where we download the instance-seg dataset from Kolena,
+then add a placeholder embedding (a zero-filled array):
+```
+from kolena.dataset import download_dataset
+
+dataset = "instance-seg"
+df = download_dataset(dataset)
+df_embedding = df[id_fields]
```

Comment:
unclear what the `id_fields` are in this example

Reply:
updated to `locator` as it's most common id field